src/res/translation/translation_it_IT.ts

```diff
@@ -938,12 +938,12 @@
     <message>
         <location filename="../../clientdlg.cpp" line="322"/>
         <source>Use &amp;Two Rows Mixer Panel</source>
-        <translation type="unfinished"></translation>
+        <translation>Abilita &amp;Vista su due Righe</translation>
     </message>
     <message>
         <location filename="../../clientdlg.cpp" line="330"/>
         <source>&amp;Clear All Stored Solo and Mute Settings</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
It looks like there is a missing translation here. Or is it done on purpose maybe?